src/jit/simd.h

```diff
@@ -49,4 +49,14 @@ struct SIMDIntrinsicInfo
 
 #endif // FEATURE_SIMD
 
+enum SIMD_Level
+{
+    SIMD_Not_Supported,
+    SIMD_Min_Supported, // SSE2 is the minimum SIMD level on x86/x64 platforms
```

Comment:
Should you just name this `SIMD_SSE2_Supported` and put it under `_TARGET_XARCH_`, and let the other platforms define their levels?

Reply:
This makes sense, `SIMD_SSE2_Supported` is more readable.